### PR TITLE
Test playbooks

### PR DESCRIPTION
**Pull request by mdellweg**
This changes the `make test_<...>` rule so that for example composite_content_view tests do not run, when content_view tests are requested.

Also i started a small guideline about the test infrastructure.

### CODE REVIEW

**Comment by mdellweg** [comment]
I'd like your opinion here @akofink, @gregswift .

File: test/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+In `test/test_playbooks/tasks`, preconfigured tasks for individual entities / actions can be found.
+They can be reused, to create common dependent resources, as well as to manipulate the entities in the actual test.
+If the boolean variable `expected_change` is set, such a task fails if the result does not meet this expectation.
+
+The ansible inventory contains two hosts:
+
+- fixtures: This host runs locally without modification it is meant to setup (and teardown) dependent resources, only when (re-)recording.
+- tests: This host should run the actual tests. It is used to record the vcr-yaml-files, or to run isolated against those files.
+
+After writing a test playbook, `make record_<playbook name>` must be called, and the resulting vcr files (`test_playbook/fixtures/<playbook_name>-*.yml`) must be checked into git.
```
Comment (akofink) [inline test/README.md]: Maybe include "... must be called against a running Foreman or Katello server"

File: test/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Guidedeline to writing tests
```
Comment (akofink) [inline test/README.md]: I like this readme. Let's link to it from the top level Readme as well.

File: test/README.md (hunk 1 above)
Comment (gregswift) [reply test/README.md]: Ya, a better way of taking line 20 and making it less of an afterthought to line 19 would be nice.

**Pull request merged**